### PR TITLE
foreman_datacenter 1.24.1

### PR DESCRIPTION
**Pull request by Zenya**

### CODE REVIEW

**Comment by Zenya** [comment]
@evgeni hi
I decided to add older version to rpm before move to 2.0.0 (PR #4926)
so as you recommended:
I removed %{foreman_apipie_cache_plugin} 
fixed homepage in gemspec file
created new gem file
created new release

File: rel-eng/tito.props [hunk 1]
```diff
@@ -530,3 +530,8 @@ whitelist = katello
   rubygem-runcible
   rubygem-typhoeus
   rubygem-zest
+
+
+[foreman-plugins-nightly-rhel7]
```
Comment (evgeni) [inline rel-eng/tito.props:535]: please add it to the `foreman-plugins-1.24-rhel7` section instead, roughly line 333

**Comment by evgeni** [comment]
I'd like to first get it into develop in #5116, but then 2.0 and 1.24 can follow quickly :)

**Comment by evgeni** [comment]
this is quite outdated, I'll close it and follow up with what we have in #5335

**Pull request closed without merging**